GVFS/GVFS.Notifications/VFSForGit.Mac/org.vfsforgit.usernotification.plist

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+    <dict>
+        <key>EnvironmentVariables</key>
+        <dict>
+            <!--
+            Although VFSForGit.app gets launched in the GUI context of actively logged-in user, it
+            does not seem to inherit any user-specific environment variables (like user customized
+            $PATH specified in shell ~/.profile). VFSForGit needs /usr/local/bin to be in its 
+            search PATH environment, so specifying it here, along with rest of the default paths 
+            that are set by launchd.
+            -->
+            <key>PATH</key>
+            <string>/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin</string>
+        </dict>
+        <key>Label</key>
+        <string>org.vfsforgit.usernotification</string>
+        <key>ProgramArguments</key>
+        <array>
+            <string>/usr/local/vfsforgit/VFSForGit.app/Contents/MacOS/VFSForGit</string>
```

Comment:
During the last offline discussion there was a question as to wether or not we'd have to use the name `VFSForGit`

- Are we required to use this name, or can it be overridden in a plist file?
- If our App name will always be used, should we include spaces in the name like `Microsoft Outlook.app`  `Visual Studio.app` do?

Reply:
Am not able to override the name in info.plist file. Added spacing in the name - new name is `VFS For Git.app`.